File: challenges/a-very-big-sum/problem.py
#!/bin/python3

import sys

def aVeryBigSum(n, ar):
    total = 0
    for x in range(0,n):
        total += ar[x]
    return total

n = int(input().strip())
ar = list(map(int, input().strip().split(' ')))
result = aVeryBigSum(n, ar)
print(result)

File: challenges/sock-merchant/problem.rb
#!/bin/ruby

def sockMerchant(n, ar)
    d = {}
    m = 0
    ar.each do |c|
        if d[c]
            m += 1
            d.delete(c)
        else
            d[c] = true
        end
    end
    return m
end

n = gets.strip.to_i
ar = gets.strip
ar = ar.split(' ').map(&:to_i)
result = sockMerchant(n, ar)
puts result;

